internal/olm/operator/registry/index/registry_pod.go

```diff
@@ -87,6 +89,12 @@ type RegistryPod struct { //nolint:maligned
 	pod *corev1.Pod
 
 	cfg *operator.Configuration
+
+	// FBC data
+	FBCcontent  string
+	FBCdir      string
+	FBCfile     string
+	HasFBCLabel bool
```

Comment:
With this change, is there still a need for the `BundleItems` field for backwards compatibility?

It seems like we're perhaps mixing sqlite-specific registry pod concerns with FBC registry pod concerns?

I wonder if we should have separate structs for `SqliteRegistryPod` vs `FBCRegistryPod`?

Reply:
Also, some Go idiomatic naming nits:
```suggestion
	// FBC data
	FBCContent  string
	FBCDir      string
	FBCFile     string
	HasFBCLabel bool
```

Reply:
@joelanford I split RegistryPod into two different structs (`SQLiteRegistryPod` and `FBCRegistryPod`), so the code paths for these remain separate (there's `registry_pod.go` file that handles the sqlite stuff and `fbc_registry_pod.go` file in a different package for all of the FBC stuff). 

When we deprecate SQLite stuff and completely remove the commands, the `registry_pod.go` file can go away. Let me know if this is the right way go about it or if we should refactor it in a different way. I'm open to your suggestions.